universal-login-sdk/src/config/SdkConfigDefault.ts

```diff
@@ -20,7 +20,7 @@ export const SdkConfigDefault: SdkConfig = {
   ],
   notice: '',
   authorizationsObserverTick: 1000,
-  balanceObserverTick: 1000 * 5,
+  balanceObserverTick: 1000 * 8,
   erc721TokensObserverTick: 1000 * 5,
```

Comment:
Could you decrease erc721TokensObserverTick also? In my opinion, erc721Tokens are less important than erc20